packages/react/src/live-region/Announce.features.stories.tsx

```diff
@@ -36,3 +37,11 @@ export const WithDelay = () => {
 
   return <Announce delayMs={1000}>{message}</Announce>
 }
+
+export const WithCustomElement = () => {
+  return (
+    <Announce>
+      <RelativeTime date={new Date('2020-01-01T00:00:00Z')} noTitle={true} />
```

Comment:
Note, the announcement in this story will still be the default date (and not the "on ..." message) since when the story renders and gets announced the custom element hasn't updated yet.